codersdk/templates.go

```diff
@@ -246,3 +246,17 @@ type AgentStatsReportResponse struct {
 	// TxBytes is the number of transmitted bytes.
 	TxBytes int64 `json:"tx_bytes"`
 }
+
+// TemplateExamples lists example templates embedded in coder.
+func (c *Client) TemplateExamples(ctx context.Context, organizationID uuid.UUID) ([]TemplateExample, error) {
+	res, err := c.Request(ctx, http.MethodGet, fmt.Sprintf("/api/v2/organizations/%s/templates/examples", organizationID), nil)
```

Comment:
Why are template examples org-scoped? Seems like you did it so it could be `/templates/examples` but I think you should instead have `/api/v2/template-examples` or something since it's not org scoped.

Reply:
These are admin only so I felt it was better to keep it all under that same auth schema. Do you think it really impacts anything until we have multi-org?